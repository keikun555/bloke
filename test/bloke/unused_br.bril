@main(a: int, b: int): int {
  true: bool = const true;
  br true .then .else;
.then:
  ret a;
.else:
  ret b;
}
