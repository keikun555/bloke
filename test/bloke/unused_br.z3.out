And(a.0.0 == a.0.0,
    b.0.0 == b.0.0,
    And(And(a.0.1 == a.0.0, b.0.1 == b.0.0),
        And(And(true.0.0 == True),
            If(true.0.0,
               And(BRIL.RETURN.0 == IntV(a.0.1)),
               And(BRIL.RETURN.0 == IntV(b.0.1))))))
